### ISSUE TITLE
The polygon of the geometry field is not drawn from 4.9

### ISSUE THREAD

**Issue by stormyboy60** [phpmyadmin/phpmyadmin#15391]
I have a table with a geometry field that contains polygons and multipolygons like this:
POLYGON ((2.43653 49.556457.2.431825 49.557172,2.429612 ....))

On phpmyadmin 4.6.x or 4.8.x I can display the polygon:
![polygoneok](https://user-images.githubusercontent.com/53050791/61467791-84c43200-a97c-11e9-9cc2-22f5cf73b8c0.jpg)

If I update to version 4.9.x then the polygon no longer appears:
![polygonenok](https://user-images.githubusercontent.com/53050791/61467809-8d1c6d00-a97c-11e9-8c31-973f92f584fc.jpg)

I tested with mariadb 10.1 and 10.4, same problem with mysql 8.0.16.
In all cases, all SQL operations on the field are working.
Server Debian 9
Php 7.0.3

Best regards

**Comment by williamdes** [comment]
@stormyboy60 Thanks for reporting this issue.
```sql
SELECT ST_PolygonFromText('POLYGON((-71.1776585052917 42.3902909739571,-71.1776820268866 42.3903701743239,
-71.1776063012595 42.3903825660754,-71.1775826583081 42.3903033653531,-71.1776585052917 42.3902909739571))') as test_gis;
```
Source: https://postgis.net/docs/ST_PolygonFromText.html

**Comment by williamdes** [comment]
@stormyboy60 can you send test data ?

**Comment by williamdes** [comment]
Using `git bisect` the first bad commit is 9ed068aacba8744556f4f310a6c18d43768a8f3e

**Comment by williamdes** [comment]
@stormyboy60 Fixed by 5b6405b7cafbebf0cd1f51da493a43cd6f97dd31
You can try the patch if you want :)
